src/main/resources/application.properties

```diff
@@ -25,6 +25,10 @@ spring.jpa.properties.hibernate.export.envers=true
 
 spring.jpa.open-in-view=false
 
+oauth2.jwk.rsakey.id=CHANGEME
+oauth2.jwk.rsakey.private=classpath:/jwk-private.pem
```

Comment:
I do not see these files actually stored anywhere. Are these just defaults that point to non-existent files?

Reply:
These point to the files generated by the `generateJWK` task dependecies in build.gradle.kts.

Reply:
Okay, sounds good.